ide-configs/eclipse/wildfly-cleanup.xml

```diff
@@ -1,56 +1,121 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <profiles version="2">
-<profile kind="CleanUpProfile" name="Wildfly" version="2">
```

Comment:
could you remove the ide-configs changes from your PR please?

Reply:
Absolutely. Not sure how those changes were made or how they slipped into the PR. :)

Reply:
That must have been my branch being slightly out-of-date. Updated my fork and that change doesn't show anymore.